### PR TITLE
Features

### PR DESCRIPTION
**Pull request by Fanger53**
In this project, we have done the following:

- Installed Rspec

- Run rspec --init to automatically generate the spec folder and the .rspec file

- Identified the specification of each enumerable method

- Wrote tests to check if those specifications are met in form of examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,79 @@
...
+<p align="center">
+  <a href="https://github.com/Fanger53/enumerables-rspec">
+    <img src="https://res.cloudinary.com/growsurf-prod/image/upload/v1582211139/production/gnysw2objzekbagrqiax.png" alt="Logo" width="350" height="70">
+  </a>
+</p>
+
+<h1 align="center">Enumerables Rspec</h1>
+
+<p align="center">
+  <strong>Project created by  and David Leonardo Castillo</strong>
```
Comment (Georjane) [inline README.md]: Also, I think you had a typo error here, please do well to correct that too.
```suggestion
  <strong>Project created by David Leonardo Castillo</strong>
```

**Review by Georjane — CHANGES_REQUESTED** [review]
Hi Team @janis-jenny and @Fanger53 

You have done a great job with RSpec testing. You tested all the enumerable methods and included both positive and negative test cases, nicely done!

However, you just need to add a description to your PR to get the project approved and see the comment about the Readme file below

File: enumerables.rb [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+    new_arr = []
+    my_each do |n|
+      new_arr << n if yield(n)
+    end
+    new_arr
+  end
+
+  # my_all Method
+  def my_all?(type = nil)
```
Comment (layzrd) [inline enumerables.rb]: Refactor `my_all?` method, we can extract code fragments to another method.

**Review by layzrd — APPROVED** [review]
### Status: Approved 🙌 🎇
Hi @Fanger53 and @janis-jenny, Congratulations 🎊 , you did very well on this project.    Kindly, it could better to do the following optional enhancements.
[Optional]

- Kindly refactor the `my_all?` method. We can extract code fragments to another function.
- Thoughtfully don't overuse rubocop disable cops' comments.

 Happy coding 👨‍💻
[ Abdulaziz Redi Ali](https://azizali.ml/)

**Pull request merged**